gbpservice/nfp/configurator/agents/generic_config.py

```diff
@@ -268,8 +279,9 @@ def _process_event(self, ev):
             if (resource_data.get('periodicity') == gen_cfg_const.INITIAL and
                     result == common_const.SUCCESS):
                 notification_data = self._prepare_notification_data(ev, result)
-                self.sc.poll_event_done(ev)
+                # self.sc.poll_event_done(ev)
                 self.notify._notification(notification_data)
+                return STOP_POLLING
```

Comment:
long time back "poll_event_done" was disabled, orchestrator moved to STOP_POLLING but configurator didnt. this change is not for perf.